### DISCUSSION TITLE
Stuck at Downloading World Data

### DISCUSSION THREAD

**Discussion by DraigNET** [OwlGamingCommunity/V#39] [Support]
Seem to be stuck at this screen with some annoying Halloween type music. Any ideas?
![image](https://github.com/OwlGamingCommunity/V/assets/66184380/00278831-3d15-4bbb-833f-87d962fce6ce)

**Comment by DraigNET** [comment]
Solved.

**Reply by iacobcatalin123** [reply]
please write the solution not "solved"

**Reply by DraigNET** [reply]
The solution was to get hold of the version of the script that actually works and not this broken piece of crap.

**Reply by iacobcatalin123** [reply]
What do you mean? Is owlGaming V not good?

**Reply by DraigNET** [reply]
This script doesn't work. I have the final version that wasn't released to the public which works.

**Reply by countree103103** [reply]
Where can I get that version?
